modules/self-managed-node-group/main.tf

```diff
@@ -459,10 +459,6 @@ resource "aws_security_group" "this" {
 
   tags = merge(
     var.tags,
-    {
-      "Name"                                      = local.security_group_name
```

Comment:
please add back in the `"Name"` tag, then we should be good to go with this one

Reply:
Should be all set now.